src/rmDefinitionSearch.js

```diff
@@ -1,16 +1,17 @@
 import { getServer } from "./server.js";
 import axios from 'axios';
 
-const QueryURLTemplate =  "/recordm/recordm/definitions/search/name/__DEF_NAME__?from=__FROM__&size=__SIZE__&q=__QUERY__"
+const QueryURLTemplate =  "/recordm/recordm/definitions/search/name/__DEF_NAME__?from=__FROM__&size=__SIZE__&q=__QUERY__&tz=__TIMEZONE__"
 const ResultsURLTemplate = "#/definitions/__DEF_ID__/q=__QUERY__"
 
-const rmDefinitionSearch = async function (definitionName, query="*", from=0, size=0, sort="", ascending="") {
+const rmDefinitionSearch = async function (definitionName, query="*", from=0, size=0, sort="", ascending="", tz="") {
```

Comment:
maybe `timezone=` instead of `tz`? Let's try to make this consistent.

Reply:
Made it consistent